edufinder/edufinder/rest_api/management/question_prioritization.py

```diff
@@ -0,0 +1,131 @@
+from edufinder.rest_api.models import UserAnswer, Answer, AnswerChoice, Question, Education
+import pandas as pd
+import numpy as np
+from math import log, e
+from django.core.cache import cache
+
+def get_question_tree():
+    return cache.get('question_tree') or create_question_tree()
+
+def create_question_tree():
+    dataset = fetch_data()
+    if dataset is None:
+        return
+    questions = dataset.columns.tolist()[:-1]
+    tree = create_branch(dataset, questions)
+    cache.set('question_tree', tree)
+    return tree
+
+def create_branch(dataset, questions, depth = 1):
+    left = len(questions)
+    data_count = len(dataset.index)
+    if left == 0 or data_count == 0 or depth > 13:
```

Comment:
Hvorfor 13?

Reply:
Hvorfor ikke?

Reply:
sum x=0 to 13 3^x Derfor

Reply:
Space stiger med (sum x=0 to (magic number) 3^x)